### ISSUE TITLE
[Read | LearningMaterials] Enhance CourseProgress read model with handling TaskWasUncompleted 

### ISSUE THREAD

**Issue by nowakprojects** [CodersCrew/coderscamp#229]
**Dependencies (blockers)**
- [x] Make sure that #228 is done (must be merged before)
- [x] CourseProgress is already done

**Subtasks**
- [ ] Handle in `CourseProgressReadModule` event TaskWasUncompleted
- [ ] Update learningMaterialsCompletedTasks (decrease by one) CourseProgress entity on TaskWasUncompleted

**Comment by KlaraGajaszek** [comment]
@nowakprojects I should also write tests in this task in read>course-progress, yea?

**Comment by nowakprojects** [comment]
It'd be great :) And following my beliefs - must have. 
You will find examples how to test read slices here: 
https://github.com/CodersCrew/coderscamp/blob/main/packages/api/src/module/read/learning-materials/learning-materials.read-module.spec.ts
